pygmt/tests/test_info.py

```diff
@@ -115,6 +127,17 @@ def test_info_per_column():
     )
 
 
+def test_info_per_column_with_time_inputs():
+    """
+    Make sure the per_column option works with time inputs.
+    """
+    table = pd.date_range(start="2020-01-01", periods=5).to_numpy()
+    output = info(table=table, per_column=True, f="0T")
```

Comment:
Actually, do we want a proper long alias for [`f`](https://docs.generic-mapping-tools.org/6.1/gmt.html#f-full) here?

Reply:
Yes.

Reply:
Ok, will do in a separate PR.